[WFLY-20822]: Exclude Testsuite artifacts from release.

ehsavoie opened the pull request:
Excluding tests artefects from the release:
 * wildfly-ee-galleon-pack-layer-tests
 * wildfly-galleon-pack-layer-tests
 * wildfly-spec-api-test
 * wildfly-testsuite-aggregator-base
 * wildfly-testsuite-aggregator-expansion
 * wildfly-testsuite-aggregator-preview
 * wildfly-ts-build-demander-base
 * wildfly-ts-build-demander-expansion
 * wildfly-ts-build-demander-preview
 * wildfly-testsuite-shared
 * wildfly-webservices-tests-integration

Jira: https://issues.redhat.com/browse/WFLY-20822

____

<--- THIS SECTION IS AUTOMATICALLY GENERATED BY WILDFLY GITHUB BOT. ANY MANUAL CHANGES WILL BE LOST. --->

> WildFly issue links:
> * [WFLY-20816](https://issues.redhat.com/browse/WFLY-20816)

<--- END OF WILDFLY GITHUB BOT REPORT --->

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

rhusar commented on ee-feature-pack/layer-metadata-tests/pom.xml:
```diff
@@ -26,6 +26,11 @@
     <packaging>jar</packaging>
 
+    <properties>
+        <!-- Don't try to deploy the module -->
```
Just an opinion, but I find this comment redundant, since this is well known what this property does (11x).

https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html#skip

ehsavoie replied:
it is already present in our testsuite/pom.xml

ehsavoie committed:
[WFLY-20822]: Exclude Testsuite artifacts from release.

Excluding tests artefects from the release:
 * wildfly-ee-galleon-pack-layer-tests
 * wildfly-galleon-pack-layer-tests
 * wildfly-spec-api-test
 * wildfly-testsuite-aggregator-base
 * wildfly-testsuite-aggregator-expansion
 * wildfly-testsuite-aggregator-preview
 * wildfly-ts-build-demander-base
 * wildfly-ts-build-demander-expansion
 * wildfly-ts-build-demander-preview
 * wildfly-testsuite-shared
 * wildfly-webservices-tests-integration

Jira: https://issues.redhat.com/browse/WFLY-20822

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>

ehsavoie commented:
@rhusar I've updated to take the nxrm3-maven-plugin into account

ehsavoie commented:
@rhusar I've updated to take the nxrm3-maven-plugin into account

xjusko committed:
[WFLY-20816] Convert disabling of nxrm3-maven-plugin deploy to using skipNexusStagingDeployMojo

Jira: https://issues.redhat.com/browse/WFLY-20816

rhusar commented on ee-feature-pack/layer-metadata-tests/pom.xml:
```diff
@@ -26,6 +26,12 @@
 
+    <properties>
+        <!-- Don't try to deploy the module -->
+        <maven.deploy.skip>true</maven.deploy.skip>
```
Is there any reason to keep this property around, my understanding is that this is no longer consumed anywhere?

rhusar replied:
Gotcha, I should have been more specific, i.e.  "will no longer be consumed once we upgrade to jboss-parent 50 since it disables that default execution".

Nevertheless, I am still not personally convinced, I will just remove the unused property from the projects I manage to avoid confusion.

rhusar reviewed (approved):
LGTM now.
